eq-author-api/schema/typeDefs.js

```diff
@@ -73,12 +72,31 @@ type Questionnaire {
   publishStatus: PublishStatus!
   publishDetails: [PublishDetails]
   totalErrorCount: Int!
+  surveyId: String
+  previewTheme: String!
+  themes: [Theme!]!
 }
 enum HistoryEventTypes {
   system
   note
 }
 
+type Theme {
+  id: ID!
+  enabled: Boolean!
+  shortName: String!
```

Comment:
```suggestion
  shortName: ThemeShortName!
```

Reply:
Sure, good call; done now. NB we might have to add a few more themes to the enum for the more exotic ones - at the mo it's just `census`, `northernireland` and `default` in there